Sixteen750/src/main/java/org/firstinspires/ftc/sixteen750/subsystems/BrakeSubsystem.java

```diff
@@ -9,8 +9,8 @@
 @Configurable
 public class BrakeSubsystem {
 
-    public static double BRAKE_POS = 0.5; // 0.5 1.0
-    public static double GO_POS = 0.1;
+    public static double BRAKE_POS = 0.3; // 0.5 1.0
```

Comment:
Please comment as to whether these have been tested on the bot, and if so, when.